client/src/components/AutoHeightVirtualList.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { List, CellMeasurerCache } from "react-virtualized";
+
+interface AutoHeightVirtualListProps {
+  list_ref: React.RefObject<any>;
```

Comment:
Ugh, I tried with `<List>` instead of any, but `@types/react-virtualized`'s typing for `Grid` is missing the `getTotalRowsHeight` method.
![image](https://user-images.githubusercontent.com/11301919/120540784-12fe0180-c3b7-11eb-9fb5-6cc255411188.png)

Wonder if in a case like this it's worth hitting `@types/react-virtualized` with the ol'patch-package?

Reply:
Yup I ran into the same issue. I personally would rather not do any patching just for this. One workaround could be `(this.list_ref.current.Grid as any).getTotalRowsHeight();`
casting `as any` actually seem like common thing to do in typescript.. At least from what I've seen online

Reply:
That workaround looks good to me, definitely beats patching!

Reply:
Hmm, well, it might actually be a bad idea to use a flawed type. Might confuse future devs who might try and use the other `react-virtualized` methods listed in the docs that are missing from the type. I guess just leave a comment on the `any` type ref mentioning that missing fields in `@types/react-virtualized`.